test/unit/account_info/test_account_info.py

```diff
@@ -414,30 +415,24 @@ def _make_sqlite_account_info(self, env=None, last_upgrade_to_run=None):
                 last_upgrade_to_run=last_upgrade_to_run,
             )
 
-    def test_uses_default(self):
-        account_info = self._make_sqlite_account_info(
-            env={
-                'HOME': self.test_home,
-                'USERPROFILE': self.test_home,
-            }
-        )
-        actual_path = os.path.abspath(account_info.filename)
-        assert os.path.join(self.test_home, '.b2_account_info') == actual_path
```

Comment:
adjusting this test to make it pass made it moreless identical with `test_uses_xdg_config_home`, so it's removed